frontend/packages/integration-tests-cypress/README.md

```diff
@@ -0,0 +1,50 @@
+#### Getting Started
+- [What is Cypress?](https://www.youtube.com/watch?v=dr10Z-HpsCQ) (video)
+- [Cypress in a Nutshell](https://www.youtube.com/watch?v=LcGHiFnBh3Y) (video)
+
+#### Best Practices
+- We are switching over our test ids from `< ... data-test-id=".."/>` to using the Cypress preferred 
+`< ... data-test=".."/>`.  This allows us to better take advantage of certain Cypress tooling, like the 
+[Selector Playground](https://docs.cypress.io/guides/core-concepts/test-runner.html#Selector-Playground) 
+- Use [Cypress's Best Practices for Selecting Elements](https://docs.cypress.io/guides/references/best-practices.html)
+
+#### Migrating Protractor tests to Cypress
+
+When migrating a test suite from Protractor to Cypress, the following steps are recommended:
+1. Create the new test suite in Cypress
+2. If you need to create a new test id use the `data-test` attribute and the `cy.getByTestID()` helper method.
+If you need to access the legacy `data-test-id` attribute, use the `cy.getByLegacyTestID()` helper method.
+3. Remove test suite from Protractor
+
+#### Directory Structure
+```
+frontend/packages/integration-tests-cypress/
+├── cy-command-additions    <--- add commands to Cypress 'cy.' global
+│   ├── index.ts
+│   ├── nav.ts
+│   ├── project.ts
+│   ├── README.md
+│   └── selectors.ts
+├── fixtures                <--- mock data
+│   └── example.json
+├── tests                   <--- test suites
```

Comment:
`/cypress/integration-tests`?

Reply:
Not sure this is needed, feel `tests` is pretty clear.

Reply:
> Not sure this is needed, feel `tests` is pretty clear.

Let me ask another way: is there a reason to deviate from Cypress recommended directory structure?

Reply:
We can leave this if we're not renaming the parent folder to cypress (since we want to give it a specific package name).